actions_test.go

```diff
@@ -79,7 +95,7 @@ func TestAction_AddMask(t *testing.T) {
 	t.Parallel()
 
 	var b bytes.Buffer
-	a := NewWithWriter(&b)
+	a := New(OptWriter(&b))
```

Comment:
Ditto here on editorializing; I did this to prepare for just straight up deleting `NewWithWriter()`. I figured it'd make sense to leave it in for at least one commit to `main` to give people a chance to migrate to `New(opts...)`.

Reply:
Also fine